.github/workflows/extract-translation-source-files.yml

```diff
@@ -168,21 +170,23 @@ jobs:
       # Extracts the translation source files
       - name: extract translation source files
         run: |
-          cd ${{ matrix.repo }}
+          cd translations/${{ matrix.repo }}
           make extract_translations
-      # git adds only the translation source files, found in /src/i18n/transifex_input.json
+      # git adds only the translation source files, found in src/i18n/transifex_input.json
       - name: git add the translation source files
         id: add-sources
         run: |
           # set identity
           git config --global user.email "translations-bot@openedx.org"
           git config --global user.name "edx-transifex-bot"
-          # remove ${{ matrix.repo }}/.git so we don't commit a submodule
-          rm -rf ${{ matrix.repo }}/.git
+          # Change directory to translations/${{ matrix.repo }}
+          cd translations/${{ matrix.repo }}
```

Comment:
I added the `cd` so that the step `EN_DIR=$(find . -type d -regex ".+conf\/locale\/en$")` would definitely find the directory we wanted in the python job. This wasn't necessary for the javascript job, but I thought some symmetry would be useful.